File: java/src/main/java/com/example/demo/entities/Vehicle.java
package com.example.demo.entities;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class Vehicle {
    private Long Id;
    private String vehicle_name;
    private String vehicle_no;

    public Vehicle(String vehicle_name, String vehicle_no){
        this.Id = null;
        this.vehicle_name = vehicle_name;
        this.vehicle_no = vehicle_no;
    }

    public Vehicle(Long Id, Vehicle other){
        this.Id = Id;
        this.vehicle_name = other.vehicle_name;
        this.vehicle_no = other.vehicle_no;
    }
}

File: python/src/entities/Vehicle.py
class Vehicle:
    def __init__(self, vehicle_name, vehicle_no):
        self.Id = None
        self.vehicle_name = vehicle_name
        self.vehicle_no = vehicle_no

    def __init__(self, Id, other=None):
        self.Id = Id
        self.vehicle_name = other.vehicle_name
        self.vehicle_no = other.vehicle_no
            